src/render/Geometry.h

```diff
@@ -61,6 +63,12 @@ class Geometry : public QObject
         virtual bool loadFile(QString fileName, size_t maxVertexCount) = 0;
 
         //--------------------------------------------------
+        /// Mutate a geometry
+        ///
+        /// Modify some geometry data with the data in a GeometryMutator
```

Comment:
Would be good to document the action of the mutator more fully.  Specifically, `GeometryMutator` applies per-vertex changes.